### PR TITLE
feature/AT-SCE-UI_Templates_adminusers

### PR DESCRIPTION
**Pull request by FlavioAlejandro**

### CODE REVIEW

**Review by luishcastroc — APPROVED** [review]
i suggest you to install prettier, i believe we asked for this earlier, this to be consistent in the format of the code.

File: src/views/admin/adminusers.hbs [hunk 1]
```diff
@@ -0,0 +1,213 @@
+
```
Comment (at-internship) [inline src/views/admin/adminusers.hbs]: This file should be named add-users.hbs based on this folder structure https://github.com/at-internship/at-sso-ui/blob/develop/src/views/admin/user/add-user.hbs

Comment (at-internship) [reply src/views/admin/adminusers.hbs]: Please do not mix behaviors of:
index.hbs
add-user.hbs

Each view has its own purpose.

File: src/views/admin/index.hbs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="jumbotron mt-1">
+     <div class="container" style="text-align: left;">
+
+          <h1>Admin Module</h1>
+          <!--linea -->
+          <hr color="blue" size=3>
+           <h4>AT-SCE</h4>
+
+           <button type="button" class="btn btn-primary" href="/admin/user"> User</button>
```
Comment (at-internship) [inline src/views/admin/index.hbs:9]: Are we sure this route works?

File: src/views/calculator.hbs [hunk 3]
```diff
@@ -1,10 +1,265 @@
-<div class="jumbotron mt-1">
-     <div class="container" style="text-align: center;">
-
+<div class="pt-1">
```
Comment (at-internship) [inline src/views/calculator.hbs:1]: this file already exists in develop, why do we add it again?

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by at-internship — CHANGES_REQUESTED** [review]
Please review admin section and conde indent

**Pull request closed without merging**